newIDE/app/src/QuickCustomization/GameImage.module.css

```diff
@@ -0,0 +1,8 @@
+.illustrationImage {
```

Comment:
(Note that all of this could have been an inline style)

Reply:
updating this